_pages/dev/ssr/server-side-rendering-error-handling.md

```diff
@@ -0,0 +1,422 @@
+---
+title: SSR Error Handling
+feature:
+  - name: SSR Error Handling
+    spa_version: 2211.29
+    cx_version: n/a
+---
+
+SSR Error Handling functionality provided the missing behavior to how Angular engine handles errors during rendering pages on the server. It ensures the Spartacus application reacts on the encountered errors by providing default set of tools to handle them and, at the same time, giving a possibility to customize the experience. This feature is crucial for the SEO of the page and the overall UX.
+
+**Note:** Angular SSR by default doesn't ignore an error only if occurred in the APP_INITIALIZER or synchronous bootstrap of the root component.
+
+To benefit from all aspects of SSR error handling, you must do the following: 
```

Comment:
```suggestion
New Spartacus apps created since v2211.29, have the SSR Error Handling enabled by default.

If your Spartacus app was created before v2211.29, to benefit from all aspects of SSR error handling, you must do the following: 
```

Reply:
such information is included in the note in line 20:
```json
**Note:** The SSR Error Handling is available starting from Spartacus 2211.29. For new application, the feature is enabled fy default. For migrated application. the feature is disabled by default and can be enabled by using mentioned feature toggles. After expiration of the feature toggle period, the feature will be enabled by default.
```